spring-cloud-task-batch/src/main/java/org/springframework/cloud/task/batch/configuration/TaskBatchAutoConfiguration.java

```diff
@@ -35,7 +35,7 @@
  * @author Michael Minella
  */
 @Configuration
-@ConditionalOnClass({Job.class, EnableTask.class})
+@ConditionalOnBean({Job.class})
```

Comment:
While this proved the point...we should also make this conditional on a property allowing a user to turn this functionality off if they want.